packages/react/src/time-input/TimeInput.test.tsx

```diff
@@ -21,7 +21,7 @@ function ControlledTimeInput({
 	value: valueProp,
 	...props
 }: ControlledTimeInputProps) {
-	const [value, setValue] = useState<TimeValue | undefined>(valueProp);
+	const [value, setValue] = useState<TimeValue>(valueProp);
```

Comment:
This should be reverted back to 
```suggestion
	const [value, setValue] = useState<TimeValue | undefined>(valueProp);
```

Unless valueProp defaults to a default value then it needs to be typed as `TimeValue | undefined`

Reply:
Oh.. I see that you changed the TimeValue type to include `undefined` now

Reply:
@Dan503 `TimeValue` is now a union that includes `undefined`. So this state value does actually get the correct type.

Reply:
Yep, your previous comments found my bug that `undefined` wasn't valid… which is why all my examples were more verbose than they should be